deploy/complete/helm-chart/setup/values.yaml

```diff
@@ -10,6 +10,13 @@ metrics-server:
 # https://github.com/helm/charts/blob/master/stable/prometheus/README.md
 prometheus:
   enabled: true
+  extraScrapeConfigs: |
+    - job_name: 'ingress-nginx'
+      metrics_path: /metrics
+      scrape_interval: 5s
+      static_configs:
+        - targets:
+          - mushop-utils-ingress-nginx-controller-metrics:9913
```

Comment:
is it possible to set this target dynamically? I _really really_ dislike having any hard-coded references to the namespace in which this is installed

Reply:
I know. I need to change on the TF as well.

Reply:
Fixed